ONSdigital/dp-api-tests
# Allow automation of api-tests

### What

- Create a docker-compose project to run api tests

### How to review

- Follow instructions in automation README

High Level:

- `install docker and docker compose`
- Configure docker to have the resources allocated on the README image
- `cd dp-api-tests/automation`
- `docker-compose build` (takes several minutes)
- In your .env file, set API_TESTS_BRANCH to be` feature/automation`
- Run `./end-to-end-test.sh`
- Verify the tests pass

### Who can review

Anyone
